docs/sql-guide/functions/function-datetimeadd.md

```diff
@@ -0,0 +1,105 @@
+---
+title: DATETIMEADD()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# DATETIMEADD() function
+
+The `DATETIMEADD()` function adds a time duration specified using time unit and integer value to a date-time and then returns the results.
+
+## Syntax
+
+```
+DATETIMEADD(timeunit, int_expr, date-time)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| timeunit | string | String value that specifies the time unit for the duration to be added to the target date-time. | Yes | [Time units](#additional-information).|
+| int_expr | integer | Integer value specified as literal or expression to be added as time duration to the target date-time. | Yes | |
+| date-time | timestamp | The target date-time specified as literal or expression the time duration is added to. | Yes | |
+
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| timestamp | Returns date-time added to the specified time duration. |
+
+## Additional information
+
+### timeunit
+
+{% include /sql-guide/datetimeadd-function-timeunit-table.md %}
+
+## Examples
+
+### Add 62 years to date of birth to derive retirement date. 
+Here DATETIMEADD() is used to do a popular date arithmetic.
+
+```sql
+create table demo
+    (_id id, dob timestamp timeunit 's');
+
+insert into demo(_id, dob)
+    values (1, '1970-01-01T00:00:00Z');
+insert into demo(_id, dob)
+    values (2, '1980-01-01T00:00:00Z');
+insert into demo(_id, dob)
+    values (3, '1990-01-01T00:00:00Z');
+insert into demo(_id, dob)
+    values (4, '2000-01-01T00:00:00Z');
+
+select _id, dob, datetimeadd('yy',62,dob) as retirement_dt from demo;
+
+ _id | dob                           | retirement_dt
+-----+-------------------------------+-------------------------------
+   1 | 1970-01-01 00:00:00 +0000 UTC | 2032-01-01 00:00:00 +0000 UTC
+   2 | 1980-01-01 00:00:00 +0000 UTC | 2042-01-01 00:00:00 +0000 UTC
+   3 | 1990-01-01 00:00:00 +0000 UTC | 2052-01-01 00:00:00 +0000 UTC
+   4 | 2000-01-01 00:00:00 +0000 UTC | 2062-01-01 00:00:00 +0000 UTC
+```
+
+### Add 24 hours wait period to last run date to derive the next run time.
+Adding 24 hours can be coded in multiple possible ways as shown below:
```

Comment:
This information should be moved to a heading beneath "Additional information" because it's not just relevant to this example.

You'd do it something like this:

```
### Timeunit conversion table

| Timeunit | seconds | minutes | hours | days |
|---|---|---|---|
| day | 86400 | 1440 | 24 | 1`|
| hour | 3600 | 60 | 1 |  24 | 
| minute | 60 | 1 |   |   |
| second |  1 | 60 | 3600 | 86400  |
```